File: Exercice1.cpp
#include <iostream>

int main() {
    for (int i = 1; i <= 100; i++) {
        if (i % 3 == 0) {
            std::cout << "Hello";
        }
        if (i % 5 == 0) {
            std::cout << "World";
        }
        if (i % 7 == 0) {
            std::cout << "Yoo";
        }
        if (i % 3 != 0 && i % 5 != 0 && i % 7 != 0) {
            std::cout << i;
        }
        std::cout << std::endl;
    }
    return 0;
}

File: Exercice1.py
for i in range(1, 101):
    if i % 3 == 0:
        print("Hello", end="")
    if i % 5 == 0:
        print("World", end="")
    if i % 7 == 0:
        print("Yoo", end="")
    if i % 3 != 0 and i % 5 != 0 and i % 7 != 0:
        print(i, end="")
    print()
